File: dotnet/cafdemalihapa/Directories/Hosting/Feature/FrontEnd/FrontEndDirectory.cs
namespace cafdemalihapa.Directories.Hosting.Feature.FrontEnd
{
    public class FrontEndDirectory()
    {
        public string GetName()
        {
            return "fend";
        }
        public string GetPath()
        {
            var directoryName = GetName();
            var featureDirectoryPath = FeatureDirectory.GetPath();
            var frontEndDirectoryPath = Path.Combine(featureDirectoryPath, directoryName);
            return frontEndDirectoryPath;
        }
        public void Create()
        {
            var path = GetPath();
            if(!Directory.Exists(path))
            {
                Directory.CreateDirectory(path);
            }
        }
    }
}

File: dotnet/cafdemvers/zzq-cafdemwimu/cafdemwimu.console/Directories/Hosting/Feature/FrontEnd/FrontEndDirectory.fs
namespace cafdemwimu.console.Directories.Hosting.Feature.FrontEnd

open System.IO
open cafdemwimu.console.Directories.Hosting.Feature

type FrontEndDirectory() =
    member _.GetName() = "fend"

    member this.GetPath() =
        let directoryName = this.GetName()
        let featureDirectoryPath = FeatureDirectory.GetPath()
        let frontEndDirectoryPath = Path.Combine(featureDirectoryPath, directoryName)
        frontEndDirectoryPath

    member this.Create() =
        let path = this.GetPath()
        if not (Directory.Exists(path)) then
            Directory.CreateDirectory(path) |> ignore
